lc-hax/Scripts/Patches/NoRefreshDelayPatch.cs

```diff
@@ -1,17 +1,10 @@
-using System.Reflection.Emit;
-using System.Collections.Generic;
+#pragma warning disable IDE1006
+
 using HarmonyLib;
 
 namespace Hax;
 
 [HarmonyPatch(typeof(SteamLobbyManager), nameof(SteamLobbyManager.RefreshServerListButton))]
 class NoRefreshDelayPatch {
-    static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruction> instructions) {
-        bool found = false;
-
-        foreach (CodeInstruction instruction in instructions) {
-            if (instruction.opcode == OpCodes.Ldarg_0) found = true;
-            if (found) yield return instruction;
-        }
-    }
+    static void Prefix(ref float ___refreshServerListTimer) => ___refreshServerListTimer = 999f;
```

Comment:
This is so much cleaner, thank you. We can use `MaxValue` here.

```suggestion
    static void Prefix(ref float ___refreshServerListTimer) => ___refreshServerListTimer = float.MaxValue;
```

Reply:
not a good idea since the variable gets incremented continuously by a timer

Reply:
You're right, I missed that. Let's change it to 1.0f then.